pkg/db/db_local/start_services.go

```diff
@@ -411,12 +414,9 @@ func updateDatabaseNameInRunningInfo(ctx context.Context, databaseName string) (
 func createCmd(ctx context.Context, port int, listenAddresses string) *exec.Cmd {
 	postgresCmd := exec.Command(
 		getPostgresBinaryExecutablePath(),
-		// by this time, we are sure that the port if free to listen to
+		// by this time, we are sure that the port is free to listen to
 		"-p", fmt.Sprint(port),
-		"-c", fmt.Sprintf("listen_addresses=\"%s\"", listenAddresses),
-		// NOTE: If quoted, the application name includes the quotes. Worried about
-		// having spaces in the APPNAME, but leaving it unquoted since currently
-		// the APPNAME is hardcoded to be steampipe.
+		"-c", fmt.Sprintf("listen_addresses=%s", listenAddresses),
```

Comment:
There's no need to quote arguments in this context as the quote usually passed on the command line are interpreted by the shell, and we have no shell here.